occurrence-download/src/main/java/org/gbif/occurrence/download/file/simplecsv/SimpleCsvDownloadAggregator.java

```diff
@@ -127,14 +123,9 @@ private void mergeResults(List<Result> results) {
   /**
    * Persists the dataset usages collected in by the datasetUsagesCollector.
    */
-  private void persistUsages(DatasetUsagesCollector datasetUsagesCollector) {
-    for (Map.Entry<UUID, Long> usage : datasetUsagesCollector.getDatasetUsages().entrySet()) {
-      DatasetOccurrenceDownloadUsage datasetOccurrenceDownloadUsage = new DatasetOccurrenceDownloadUsage();
-      datasetOccurrenceDownloadUsage.setNumberRecords(usage.getValue());
-      datasetOccurrenceDownloadUsage.setDatasetKey(usage.getKey());
-      datasetOccurrenceDownloadUsage.setDownloadKey(configuration.getDownloadKey());
-      persistUsage.apply(datasetOccurrenceDownloadUsage);
-    }
+  private void persistUsages(String downloadkey,DatasetUsagesCollector datasetUsagesCollector) {
```

Comment:
maybe we do not need this method, change it to:
occurrenceDownloadService.createUsages(downloadkey, datasetUsagesCollector.getDatasetUsages());